src/main.zig

```diff
@@ -7143,6 +7143,18 @@ fn accessLibPath(
 ) !bool {
     const sep = fs.path.sep_str;
 
+    if (mem.startsWith(u8, lib_name, ":")) colon: {
+        try test_path.writer().print("{s}" ++ sep ++ "{s}", .{ lib_dir_path, lib_name[1..] });
+        try checked_paths.writer().print("\n  {s}", .{test_path.items});
+        fs.cwd().access(test_path.items, .{}) catch |err| switch (err) {
+            error.FileNotFound => break :colon,
+            else => |e| fatal("unable to search for library '{s}': {s}", .{
+                test_path.items, @errorName(e),
+            }),
+        };
+        return true;
+    }
```

Comment:
when there is a colon, I don't think you actually want to do a file system access, do you?

Reply:
that's right, this access is totally unnecessary since the linker will already do this job, no need to do it twice.